index.html

```diff
@@ -1,317 +1,375 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-WXBV13V4WN"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+      function gtag(){window.dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'G-WXBV13V4WN');
+    </script>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css" />
+    <title>Ellis Portfolio</title>
+  </head>
+  <body>
 
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
-    <link href="css/style.css" rel="stylesheet">
-
-
-    <title>Ellis Ng'ona</title>
-
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"></script>
-  <script>
-
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
+  <div id="menuId" class="myMenu">
```

Comment:
- [ ] Your mobile menu working correctly however, we actually want your design to closely match the provided mobile menu [Figma template](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A546&scaling=min-zoom&page-id=0%3A1). Your font sizes and spacing (margins/paddings) do not match the provided design. Kindly ensure to improve upon the mentioned aspect in order to enhance the UI of your mobile menu

**Your design (mobile menu):**

![image](https://user-images.githubusercontent.com/12745474/128450836-8e290822-0f31-4fed-af25-df0c372150db.png)


**The original design (mobile menu):**

![image](https://user-images.githubusercontent.com/12745474/128450931-9746a70e-007d-4d78-ad45-3c388e75e0cf.png)